targets/aarch64.json

```diff
@@ -0,0 +1,23 @@
+{
+	"build-tags": ["aarch64","baremetal"],
+	"goarch":"arm64",
```

Comment:
Please add `goos` and set it to `linux`. For an explanation, see https://github.com/tinygo-org/tinygo/wiki/Adding-a-new-board#build-tags